File: CustomDrawing/CustomDrawing/ViewController.m
//
//  ViewController.m
//  CustomDrawing
//
//  Created by LiuScott on 15-3-26.
//  Copyright (c) 2015年 LiuScott. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property UIImage * anImage;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    [self drawRect: CGRectMake(0, 0, 100, 100)];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void) drawRect: (CGRect) rect {
    NSString *imagePath = [[NSBundle mainBundle] pathForResource:@"swipe.png" ofType:@"png"];
    UIImage *myImageObj = [[UIImage alloc] initWithContentsOfFile:imagePath];
    self.anImage = myImageObj;
    [self.anImage drawAtPoint:CGPointMake(10, 10)];
}


@end

File: ImageViewTest/ImageViewTest/ViewController.swift
//
//  ViewController.swift
//  ImageViewTest
//
//  Created by LiuScott on 15-3-31.
//  Copyright (c) 2015年 LiuScott. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    
    
    @IBOutlet var backgroundView: UIImageView?
    @IBOutlet var myLabel: UILabel?
    
    var imageList: [String] = ["Palms.jpg", "Mountain.jpg", "NewYork.jpg"]
    var nameList: [String] = ["Beach Vaction", "Hike", "Big City"]
    let maxImages = 2
    var imageIndex = 0
    
    

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        backgroundView?.image = UIImage(named: "Palms.jpg")
        myLabel?.text = nameList[0];
    }

    @IBAction func nextIamge() {
        if imageIndex > maxImages {
            imageIndex = 0;
        }
        backgroundView?.image = UIImage(named: imageList[imageIndex]);
        myLabel?.text = nameList[imageIndex];
        imageIndex++
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

